docs/tutorials/services/accessing-and-moving-robot-arm.md

```diff
@@ -11,7 +11,12 @@ tags: ["arm", "motion", "services"]
 aliases:
   - "/tutorials/motion/accessing-and-moving-robot-arm"
   - "/tutorials/motion/"
-# SMEs: William S.
+authors: []
+languages: [ "python", "go" ]
+viamresources: [ "arm" ]
+level: "Intermediate"
```

Comment:
```suggestion
level: "Beginner"
```
It's the first of the series and doesn't assume prior knowledge. Might well be easier than several of the "intermediate" tutorials here.